config/locales/en/errors.yml

```diff
@@ -651,6 +657,67 @@ en:
           attributes:
             business_type:
               inclusion: Select a business type
+        steps/income/businesses_summary_form:
+          attributes:
+            add_business:
+              inclusion: Select yes if you want to add another business
+        steps/income/businesses_form:
+          attributes:
+            trading_name: 
+              blank: Enter the trading name of the business 
+            address:
+              address_line_one:
+                blank: Enter address line 1, typically the building and street
+              city:
+                blank: Enter town or city
+              country:
+                blank:  Enter country
+              postcode:
+                blank: Enter postcode
```

Comment:
<img width="820" alt="Screenshot 2024-07-05 at 11 24 27" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/44b45c14-2582-4871-b348-5f3412e60601">
These error messages aren't appearing